cosmosdb/types.bal

```diff
@@ -21,6 +21,7 @@ import ballerinax/'client.config;
 @display {label: "Connection Config"}
 public type ManagementClientConfig record {|
     *config:ConnectionConfig;
+    # 
```

Comment:
```suggestion
    # Excluded configurations related to client authentication 
```

Reply:
This is the record. We dont expect the user to give any value for `auth`. But user has to give `primaryKeyOrResourceToken`

```ballerina
public type ManagementClientConfig record {|
    *config:ConnectionConfig;
    # 
    never auth?;
    # Base URL of the Azure Cosmos DB account
    string baseUrl;
    # The token used to make the request call authorized
    @display{
        label: "",
        kind: "password"
    }
    string primaryKeyOrResourceToken;
    # The HTTP version understood by the client
    http:HttpVersion httpVersion = http:HTTP_1_1;
|};
```

Reply:
Fixed via [a76df57](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/pull/90/commits/a76df577c92f45595ec21808e13afbb5412f9e06)